Podcast/CardTableViewCell.swift

```diff
@@ -81,19 +61,14 @@ class CardTableViewCell: UITableViewCell {
     var episodeNameLabel: UILabel!
     var dateTimeLabel: UILabel!
     var descriptionLabel: UILabel!
-    var recommendedButton: RecommendButton!
-    var bookmarkButton: BookmarkButton!
     var seperator: UIView!
     var podcastImageView: ImageView!
-    var lineSeperator: UIView!
     var topLineSeperator: UIView!
-    var moreButton: MoreButton!
-    var playButton: PlayButton!
     var contextLabel: UILabel!
     var contextImages: [ImageView] = []
     var contextView: UIView! //view for upper context bar of feed cell
     var mainView: UIView! //main view
-    var bottomView: UIView! //bottom bar view with buttons
+    var utilityButtonBarView: EpisodeUtilityButtonBarView! //bottom bar view with buttons
```

Comment:
Why not call this episodeButtonBar, utilityButtonBarView might not be clear initially what it contains other than it is reused buttons.

Reply:
Fair I can change it to episodeUtilityButtonBarReview